src/main/resources/templates/help.html

```diff
@@ -15,26 +17,21 @@
 <body>
 <div class="content">
   <header>
-    <img src="images.png" alt="Logo">
+    <img src="../static/images/images.png" alt="Logo">
```

Comment:
Moved image and CSS files being referenced to correct /resources/static directory. [Reference](https://www.baeldung.com/spring-mvc-static-resources#spring-boot)